pyresample/geometry.py

```diff
@@ -38,11 +38,11 @@
 from pyresample.boundary import AreaDefBoundary, Boundary, SimpleBoundary
 from pyresample.utils import (
     check_slice_orientation,
-    get_geostationary_height,
     load_cf_area,
     proj4_dict_to_str,
     proj4_radius_parameters,
 )
+from pyresample.utils.proj4 import get_geostationary_height
```

Comment:
Just curious, why the import change?

Reply:
I think it's more explicit this way

Reply:
Sure, but maybe less consistent? Maybe all the `utils` should be explicit and we gret of the convenience import in `utils/__init__.py`?